content/articles/how-to-solve-initial-value-problem-using-ode45-in-matlab/index.md

```diff
@@ -0,0 +1,165 @@
+
+In this case, the `F` is the function for both *t* and *y*, which we must provide. This function takes in two variables, that is, the independent variable `t` and dependent variable `y` and returns a vector $\frac{dy}{dt}$. Since we want to look at the single variable equations, the output $\frac{dy}{dt}$ will be a scalar. Single variables are equations with single variables.
+
+ODE45 gives `t` and `y` as the arguments passed on to the function `fname`. Thus, we pass the initial conditions `t0`, `t_end` and the initial solution `y0` to the function.
+
+We now want to see how this works given an equation $\frac{dy}{dt}=-2 * t * y$
+
+### Matlab code
+
+We first define the initial conditions.
+
+```matlab
+%solving ODE-IVP using matlab's ODE45
+t0=0;    
+y0=1;
+tEnd=5;
+```
+
+We will now consider our ODE syntax.
+
+```Matlab
+%solving using ODE45
+[tsol, ysol]=ode45(@(t,y) firstODEfun(t,y), [t0 tEnd], y0);
+```
+
+`t` and `y` are the variables passed to the function `ode45` and then call the function `firstODEfun`. Note, the arrangement of `t` and `y` is essential. It has to be `t` and then `y` not the other way round. It is a mistake commonly made by many students. After this, we need to define our function `firstODEfun`. It should be on a separate script. The code below defines our function.
+
+```matlab
+function dy = firstODEfun(t,y)
+dy= -2*t*y;
+```
+
+The `dy` is the defined function.
+
+```Matlab
+plot(tsol, ysol)
+```
+
+Now we save this script and plot the output of our function.
+
+![our function plot](ode_one.png)
+
+### Common errors when doing IVP
+
+Let's make a small mistake in the `firstODEfun` by renaming the `t` in the `dy` function as `t1`. Then, when we run our program, we get an error, as displayed in the command window.
+
+![error defination](ode_two.png)
+
+The error indicates that the `t1` is undefined since the input variable was `t`. Keep in mind that our `t` and `y` are dummy variables. It means that they don't have any specification name, but you can use any name. The only thing that you should note is to use a similar name for the input and, in the definition, your function `dy`. Then, it won't be a problem. Also, as you use a different name, the order of the variable must remain, independent variable followed by the independent.
+
+### Plug flow reactor(PFR)
+
+Plug flow reactor, also known as tabular reactor, consists of a cylindrical pipe with an opening on each end for the reactor and products to flow through. Consumption of reactors is continuous as they flow down the length of the reactors. It means that as the reaction continues, the products increases.
+
+The plug flow reactor may be figured as one long tube or several short tubes. They range in diameter from few centimetres to several meters. The choice of diameter is based on the construction cost, pumping cost, and heat transfer needed. They have a wide variety of applications in either gas or liquid phase systems.
```

Comment:
PFR needs through explanation. This explanation does not guide the reader on what the problem is.